.env

```diff
@@ -1 +1 @@
-REACT_APP_JWT_SECRET=bhailang
\ No newline at end of file
+REACT_APP_JWT_SECRET=bhailang123
```

Comment:
You should not upload .env file on your github, refer to the article if you need some reference to remove
https://daily-dev-tips.com/posts/removing-a-env-file-from-git-history/

Reply:
ohh by mistake I would have uploaded it